### PR TITLE
chore: update KEGG to version 100

### PR DESCRIPTION
**Pull request by edkerk**
### Main improvements in this PR:
- feat:
  - update KEGG MAT files to KEGG version 100
  - `checkInstallation` has `develMode`, when set to _true_ it also tests CD-HIT and MAFFT binaries, otherwise these are skipped by default
  - only continue distributing pretrained HMM sets with CD-HIT sequence identity threshold of 90%, see #394 
- refactor:
  - run MAFFT and CD-HIT via WSL on Windows

**Note: the new HMM files will be attached with the RAVEN 2.6.0 release, and while the URL in `getKEGGModelForOrganism` is already modified, the files will only become available upon release. Developers can already get the files from https://github.com/SysBioChalmers/RAVEN/releases/tag/untagged-1ebb6d1b085aa1f7fb57**

**I hereby confirm that I have:**

- [x] Tested my code on my own machine
- [x] Followed the [development guidelines](https://github.com/SysBioChalmers/RAVEN/wiki/DevGuidelines).
- [x] Selected `devel` as a target branch
- [ ] If needed, asked first in the [Gitter chat room](https://gitter.im/SysBioChalmers/RAVEN) about this PR

### CODE REVIEW

File: doc/external/kegg/getKEGGModelForOrganism.html [hunk 1]
```diff
@@ -415,1042 +390,953 @@ <h2><a name="_source"></a>SOURCE CODE <a href="#_top"><img alt="^" border="0" sr
...
+0570     <span class="keyword">end</span>
+0571     missingAligned=missingAligned(randperm(RandStream.create(<span class="string">'mrg32k3a'</span>,<span class="string">'Seed'</span>,cputime()),numel(missingAligned)));
+0572     tmpFile=tempname;
+0573     <span class="comment">%On Windows, paths need to be translated to Unix before parsing it to WSL</span>
+0574     <span class="keyword">if</span> ispc
+0575         wslPath.tmpFile=<a href="getWSLpath.html" class="code" title="function path=getWSLpath(path)">getWSLpath</a>(tmpFile);
+0576         <span class="comment">%mafft has problems writing to terminal (/dev/stderr) when running</span>
+0577         <span class="comment">%on WSL via MATLAB, instead write and read progress file</span>
+0578         mafftOutput = tempname;
+0579         wslPath.mafftOutput=<a href="getWSLpath.html" class="code" title="function path=getWSLpath(path)">getWSLpath</a>(mafftOutput));
```
Comment (simas232) [inline doc/external/kegg/getKEGGModelForOrganism.html]:
```suggestion
0579         wslPath.mafftOutput=<a href="getWSLpath.html" class="code" title="function path=getWSLpath(path)">getWSLpath</a>(mafftOutput);
```

File: doc/external/kegg/getKEGGModelForOrganism.html [hunk 2]
```diff
@@ -415,1042 +390,953 @@ <h2><a name="_source"></a>SOURCE CODE <a href="#_top"><img alt="^" border="0" sr
...
+0580         wslPath.mafft=<a href="getWSLpath.html" class="code" title="function path=getWSLpath(path)">getWSLpath</a>(fullfile(ravenPath,<span class="string">'software'</span>,<span class="string">'mafft'</span>,<span class="string">'mafft-linux64'</span>,<span class="string">'mafft.bat'</span>));
+0581         wslPath.cdhit=<a href="getWSLpath.html" class="code" title="function path=getWSLpath(path)">getWSLpath</a>(fullfile(ravenPath,<span class="string">'software'</span>,<span class="string">'cd-hit'</span>,<span class="string">'cd-hit'</span>);
```
Comment (simas232) [inline doc/external/kegg/getKEGGModelForOrganism.html]:
```suggestion
0581         wslPath.cdhit=<a href="getWSLpath.html" class="code" title="function path=getWSLpath(path)">getWSLpath</a>(fullfile(ravenPath,<span class="string">'software'</span>,<span class="string">'cd-hit'</span>,<span class="string">'cd-hit'</span>;
```

File: external/kegg/getKEGGModelForOrganism.m [hunk 3]
```diff
@@ -595,16 +564,23 @@
...
-    fastaFiles=listFiles(fullfile(dataDir,'fasta','*.fa'));
-    %Align all sequences using MAFFT
+    tmpFile=tempname;
+    %On Windows, paths need to be translated to Unix before parsing it to WSL
+    if ispc
+        wslPath.tmpFile=getWSLpath(tmpFile);
+        %mafft has problems writing to terminal (/dev/stderr) when running
+        %on WSL via MATLAB, instead write and read progress file
+        mafftOutput = tempname;
+        wslPath.mafftOutput=getWSLpath(mafftOutput));
```
Comment (simas232) [inline external/kegg/getKEGGModelForOrganism.m:579]:
```suggestion
        wslPath.mafftOutput=getWSLpath(mafftOutput);
```

File: external/kegg/getKEGGModelForOrganism.m [hunk 4]
```diff
@@ -595,16 +564,23 @@
...
+        wslPath.mafft=getWSLpath(fullfile(ravenPath,'software','mafft','mafft-linux64','mafft.bat'));
+        wslPath.cdhit=getWSLpath(fullfile(ravenPath,'software','cd-hit','cd-hit');
```
Comment (simas232) [inline external/kegg/getKEGGModelForOrganism.m:581]:
```suggestion
        wslPath.cdhit=getWSLpath(fullfile(ravenPath,'software','cd-hit','cd-hit'));
```

**Review by simas232 — CHANGES_REQUESTED** [review]
In addition to the suggestions, please make the following changes to `tutorial5`:

- [x] line 12: change to "euk90_kegg100"
- [ ] line 24: change rxn count to 1590 and met count to 1571
- [ ] line 25: change gene count to 837
- [ ] run `updateDocumentation` after applying these changes

Everything else works perfectly in my machine. Great implementations!

File: external/kegg/getKEGGModelForOrganism.m (hunk 4 above)
Comment (edkerk) [reply external/kegg/getKEGGModelForOrganism.m]: Fixed in [ce329ba](https://github.com/SysBioChalmers/RAVEN/pull/395/commits/ce329ba245412b7f7f0868c985efb1a39a61a2ea)

File: external/kegg/getKEGGModelForOrganism.m (hunk 3 above)
Comment (edkerk) [reply external/kegg/getKEGGModelForOrganism.m]: Fixed in [ce329ba](https://github.com/SysBioChalmers/RAVEN/pull/395/commits/ce329ba245412b7f7f0868c985efb1a39a61a2ea)

File: doc/external/kegg/getKEGGModelForOrganism.html (hunk 2 above)
Comment (edkerk) [reply doc/external/kegg/getKEGGModelForOrganism.html]: Fixed in [fe3daa4](https://github.com/SysBioChalmers/RAVEN/pull/395/commits/fe3daa49f3c6b699b8ff204075c0d6cf04be496a)

File: doc/external/kegg/getKEGGModelForOrganism.html (hunk 1 above)
Comment (edkerk) [reply doc/external/kegg/getKEGGModelForOrganism.html]: Fixed in [fe3daa4](https://github.com/SysBioChalmers/RAVEN/pull/395/commits/fe3daa49f3c6b699b8ff204075c0d6cf04be496a)

**Review by simas232 — APPROVED** [review]

**Pull request merged**